frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -307,6 +307,9 @@ string.GitMachete.EnhancedGraphTable.duplicated-branches-text=The following bran
 string.GitMachete.EnhancedGraphTable.skipped-branches-text=Some branches listed in machete file do not exist: {0}
 action.GitMachete.EnhancedGraphTable.branch-layout-write-failure=Writing new branch layout failed
 
+action.GitMachete.EnhancedGraphTable.non-managed-branch-notification.text=Branch '{0}' is not managed by Git Machete
+action.GitMachete.EnhancedGraphTable.non-managed-branch-notification.action=Slide in
```

Comment:
I wonder if it would make it less error prone, if we use enum values for notification actions instead🤔

Reply:
Does the `Slide in` action allow the user to take any action or just notifies them?